gapic-generator/templates/default/service/client/_operations.erb

```diff
@@ -12,6 +12,37 @@ class Operations
   # @private
   attr_reader :<%= service.lro_service.stub_name %>
 
+  ##
+  # Configuration for the <%= service.name %> Operations API.
+  #
+  # @yield [config] Configure the Operations client.
+  # @yieldparam config [Operations::Configuration]
+  #
+  # @return [Operations::Configuration]
+  #
+  def self.configure
+    @configure ||= Operations::Configuration.new
+    yield @configure if block_given?
+    @configure
+  end
+
+  ##
+  # Configure the <%= service.name %> Operations instance.
+  #
+  # The configuration is set to the derived mode, meaning that values can be changed,
+  # but structural changes (adding new fields, etc.) are not allowed. Structural changes
+  # should be made on {Operations.configure}.
+  #
+  # @yield [config] Configure the Operations client.
+  # @yieldparam config [Operations::Configuration]
+  #
+  # @return [Operations::Configuration]
+  #
+  def configure
```

Comment:
I wonder if calling this `configuration` would reduce the chances of a collision with an API method name, since it's a noun rather than a verb.

Reply:
Yes, I pointed this out on the #142 PR [in this comment](https://github.com/googleapis/gapic-generator-ruby/pull/142#discussion_r283573275). We do have a potential naming conflict, but I think we can work around this by allowing a client instance object to be passed to the `Client.configure` class method:

```ruby
# Create a client instance
client = Google::Cloud::Vision::V1::ImageAnnotator::Client.new
# Configure the instance defaults by passing the client to Client.configure
# instead of calling Client#configure
Google::Cloud::Vision::V1::ImageAnnotator::Client.configure client do |config|
  config.timeout = 120
end
# Alternate way of doing this:
client.class.configure client do |config|
  config.timeout = 120
end
```

Reply:
Yes, true. Not a big deal I suppose. Carry on.

Reply:
Also, I settled on the method name `configure' because that is what Rails uses. If a client also has a `configure` RPC method, it will overwrite this method.